### PR TITLE
first pr

### PR DESCRIPTION
**Pull request by bnhoule**
Onboarding pull request

This PR fixes #

### CODE REVIEW

File: extensions/integration-tests/src/onboarding/developers.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const developers: string[] = [
 	'lucyzhang929',
 	'saiavishkarsreerama',
 	'smartguest',
+	't-bhoule',
```
Comment (aasimkhan30) [inline extensions/integration-tests/src/onboarding/developers.ts:22]: Just add bnhoule instead of t-bhoule

**Review by aasimkhan30 — CHANGES_REQUESTED** [review]

File: extensions/integration-tests/src/onboarding/developers.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export const developers: string[] = [
 	'lucyzhang929',
 	'saiavishkarsreerama',
 	'smartguest',
+	'bnhoule',
```
Comment (aasimkhan30) [inline extensions/integration-tests/src/onboarding/developers.ts:22]: Put this in line 14 to maintain the alphabetical order

**Review by aasimkhan30 — CHANGES_REQUESTED** [review]

**Review by aasimkhan30 — APPROVED** [review]

**Review by abist — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Comment by Benjin** [comment]
Don't forget to give your PR a descriptive name, and to fill out the description!

**Review by kburtram — APPROVED** [review]
LGTM.  Please merge when ever you're ready.

**Pull request merged**